pyproject.toml

```diff
@@ -69,7 +69,7 @@
     "*" = ["esptool/targets/stub_flasher/1/*", "esptool/targets/stub_flasher/2/*"]
 
 [tool.setuptools.packages]
-    find = {exclude = ["ci", "flasher_stub", "test", "docs"]}
+    find = {exclude = ["ci", "flasher_stub", "test*", "docs*"]}
```

Comment:
Hello @befeleme,
thank you for your contribution!

The `flasher_stub` is no longer a part of the repository. Could you please also remove it here as a part of this PR?

Reply:
Also, I wonder - the `ci` directory also contains a `.py` file. That doesn't get included?

Reply:
It doesn't because `ci` is explicitly excluded. If there was a subfolder with `*.py` files, they would get into the distribution. But to make it safe, I can add the * there too.

Reply:
I understand, thanks. Adding `*` to `ci` as well sounds good to me.